src/core/core.tooltip.js

```diff
@@ -325,7 +326,7 @@ module.exports = function(Chart) {
 			}
 
 			if (changed && opts.custom) {
-				opts.custom.call(me, model);
+				opts.custom.call(me, model, tooltipItems, data);
```

Comment:
I would avoid passing `data` which is useless since it can be accessed via `this.data` in the callback.